DEVELOP.md

```diff
@@ -0,0 +1,9 @@
+## Testing
```

Comment:
I wonder if this belongs in a `CONTRIBUTING.md`, which seems to be the place most OSS repos point folks when they are just getting started w/ a new project.